graphite/client.go

```diff
@@ -39,7 +39,7 @@ type GraphiteClient interface {
 func NewGraphiteClient(httpClient *http.Client, baseUrl string) (GraphiteClient, error) {
 
 	if httpClient == nil {
-		httpClient = &http.Client{Timeout:time.Second*10}
+		httpClient = &http.Client{Timeout: time.Second * 10}
```

Comment:
sorry, but can we swap this two values so it looks like the ones from line 57? Number first, timeunit second
I know it is not even related with this PR...